app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -175,15 +176,16 @@ type LayoutProps = {
 
 const Layout = ({
   children,
-  review,
   status,
   analysis,
   analysisRequired,
   analysisLoadingInSidebar,
   analysisError,
   experiment,
-}: LayoutProps) =>
-  status?.launched ? (
+}: LayoutProps) => {
+  const review = useReviewCheck(experiment);
```

Comment:
Nit: since we only need `review` if `status.launched` is falsey, we _could_ revert L185-186 and update `<AppLayoutWithSidebar {...{ status, review }}>` to `<AppLayoutWithSidebar review={useReviewCheck(experiment)} {...{ status }}>`